### DISCUSSION TITLE
Is this repo dead?

### DISCUSSION THREAD

**Discussion by karl0ss** [homarr-labs/dashboard-icons#557] [General]
I see alot of pr's open, including my own, and nothing has been merged in since October?

If this is dead, maybe we need to fork off and someone else can actively maintain it? or add some other maintainers to this repo? (would be happy to help @walkxcode)

thanks for all the great work

**Comment by lammersbjorn** [comment]
Hello, the repository is still active, but unfortunately, I haven't had the opportunity to dedicate time to its maintenance. Recently, I secured a fulltime job and started my own company, consuming most of my available time. I plan to resume the project later this year, but additional maintainers would be immensely helpful. Have you thoroughly reviewed the contribution guidelines? If you have, I'd be happy to include you as a maintainer. Wishing you a happy new year!

**Reply by lammersbjorn** [reply]
@Tinyblargon, your recent contributions have been great. Are you interested in joining the maintenance team as well?

**Reply by karl0ss** [reply]
@walkxcode I would be more than happy to help maintain, I have submitted before and understand the contribution guidelines...

**Reply by Tinyblargon** [reply]
@walkxcode Sorry I already have a few FOSS projects that I help maintain. I'll still keep helping out from time to time with finding icons.

**Reply by Hyrules** [reply]
@walkxcode if you still need someone to help let me know. I would be willing to help you with the repo. I have experience with github / git and coding. You can take a look at my repos. I'm not a big contributor in a lot of repos but if you tell me how you want stuff done would be more than happy to check every submission and pr them as needed.
